lib/aur-chroot

```diff
@@ -22,6 +22,10 @@ get_conf_repo() {
     pacconf --repo "$1"
```

Comment:
https://github.com/AladW/aurutils/blob/a001f5cd2ebd053a05488e73797aa836def8632e/lib/aur-chroot#L25-L27
The above function can be avoided if we pass the additional arguments to pacconf in `get_conf_repo`.

```suggestion
     pacconf --repo "$@"
```